src/Puppeteer.ts

```diff
@@ -14,21 +14,31 @@
  * limitations under the License.
  */
 import Launcher from './Launcher';
-import type {
+import {
   LaunchOptions,
   ChromeArgOptions,
   BrowserOptions,
 } from './launcher/LaunchOptions';
-import type { ProductLauncher } from './Launcher';
+import { ProductLauncher } from './Launcher';
 import { BrowserFetcher, BrowserFetcherOptions } from './BrowserFetcher';
 import { puppeteerErrors, PuppeteerErrors } from './Errors';
-import type { ConnectionTransport } from './ConnectionTransport';
+import { ConnectionTransport } from './ConnectionTransport';
 
 import { devicesMap } from './DeviceDescriptors';
-import type { DevicesMap } from './/DeviceDescriptors';
+import { DevicesMap } from './/DeviceDescriptors';
 import { Browser } from './Browser';
-import * as QueryHandler from './QueryHandler';
+import {
+  registerCustomQueryHandler,
+  unregisterCustomQueryHandler,
+  customQueryHandlers,
+  clearQueryHandlers,
+  QueryHandler,
+} from './QueryHandler';
```

Comment:
APIExtractor also doesn't support `import * as` imports just yet, but it [is being worked on](https://github.com/microsoft/rushstack/pull/1796). I felt this change was fine to make to unblock our progress.

Reply:
ack